constants.py

```diff
@@ -45,99 +49,124 @@ class DisplayEndingType(StrEnum):
-
-# Consider the rocket to have landed if it is within 15 meters of the launch site height
-# and the speed is low.
-GROUND_ALTITUDE = 10.0  # meters
-LANDED_SPEED = 5.0  # m/s
-
+# ----------------- Standby to MotorBurn ----------------
+ACCEL_DEADBAND_METERS_PER_SECOND_SQUARED = 0.35
+"""We integrate our acceleration to get velocity, but because IMU has some noise, and other things
+like wind or being small bumps can cause this to accumulate even while the rocket is stationary, so
+we deadband the accel to prevent this."""
+
+TAKEOFF_HEIGHT_METERS = 10
+"""The height in meters that the rocket must reach before we consider it to have taken off."""
+TAKEOFF_VELOCITY_METERS_PER_SECOND = 10
+"""The velocity in meters per second that the rocket must reach before we consider it to have taken
+off."""
+
+# ---------------- MotorBurn to Coasting ----------------
+MAX_VELOCITY_THRESHOLD = 0.96
+"""Because motors can behave unpredictably near the end of their burn, we will only say that the
+motor has stopped burning if the current velocity is less than a percentage of the max velocity."""
+
+# ----------------- Freefall to Landing -----------------
+MAX_FREE_FALL_SECONDS = 300.0
+"""The maximum amount of time in seconds that the rocket can be in freefall before we consider it to
+have landed. This is to prevent the program from running indefinitely if our code never detects the
+landing of the rocket. This value accounts for the worst case scenario of the main parachute
+deploying at apogee."""
+
+GROUND_ALTITUDE_METERS = 10.0
+"""The altitude in meters that the rocket must be under before we consider it to have landed."""
+LANDED_SPEED_METERS_PER_SECOND = 5.0
+"""The speed in meters per second that the rocket must be under before we consider it to have
+landed."""
 
 # -------------------------------------------------------
 # Apogee Prediction Configuration
 # -------------------------------------------------------
 
 # This needs to be checked/changed before flights
 FLIGHT_LENGTH_SECONDS = 22.0
+"""When we do apogee prediction, we do stepwise integration of our fitted curve to predict the
+accel, velocity, and altitude curve of the rocket. This is the total time in seconds that we will
+predict for. This is a rough estimate of the time from coast state to freefall with some extra room
+for error."""
+
+INTEGRATION_TIME_STEP_SECONDS = 1.0 / 500.0
+"""This is the dt that we use for the stepwise integration of our fitted curve. It could be any
+value and just corresponds to the precision of our prediction."""
 
-INTEGRATION_TIME_STEP = EST_DATA_PACKET_SAMPLING_RATE
+GRAVITY_METERS_PER_SECOND_SQUARED = 9.798
+"""This is the standard gravity on Earth."""
 
-# This is the standard gravity on Earth, in m/s^2
-GRAVITY = 9.798
+TARGET_ALTITUDE_METERS = 1000
+"""The target altitude in meters that we want the rocket to reach. This is used with our bang-bang
+controller to determine when to extend and retract the airbrakes."""
```

Comment:
I feel like this should be in the Coast to free fall section, since it's not actually used in apogee_predictor.py

Reply:
You made me think about this for a minute and I think you're right